### PR TITLE
#163388097 Add tests for login feature

### PR DESCRIPTION
**Pull request by PaulinhoNdegwa**
### What does this PR do?
- Add tests for login actions and reducer

### Description of Task to be completed?
- Add tests for login actions and reducer

### How should this be manually tested?

- Clone the github repository
`git clone https://github.com/andela/ah-the-phoenix-frontend.git`
- Change your working directory to the project folder
`cd ah-fulldeck-frontend`
- Checkout to thebranch ft-user-profile-163388099
`git checkout ft-user-login-163388097`
- Install the required dependencies using
` npm install`
-Run tests
` ./node_modules/.bin/jest`

### Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#163388097

### CODE REVIEW

File: src/redux/actioncreators/loginActions.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ export const sociaLogin = (social_info) => dispatch => {
             toastr.success("Welcome! ", "Success login");
         })
         .catch(error => {
+            console.log(error.request.response
```
Comment (ezrqnkemboi) [inline src/redux/actioncreators/loginActions.js]: Remove this console.

File: src/views/editProfile.js [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+import Loader from '../components/Loader/Loader';
 
 export class EditProfile extends Component {
+  componentDidMount() {
+    const { getProfile } = this.props
+    getProfile()
+  }
   render() {
+    const { isFetching } = this.props;
+    if (isFetching === true) {
```
Comment (ezrqnkemboi) [inline src/views/editProfile.js]: Good work rendering a loader.

File: src/redux/actioncreators/loginActions.js (hunk 1 above)
Comment (PaulinhoNdegwa) [reply src/redux/actioncreators/loginActions.js]: Fixed!

**Comment by PaulinhoNdegwa** [comment]
> Please check on the feedback and update them.

Done

**Pull request merged**